src/components/Badge/index.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+import styles from './Badge.module.css';
+
+import { IBadgeProps } from './IBadge.d';
+const Badge: FC<IBadgeProps> = ({
+  children = 'Badge',
+  color = 'primary',
+  className,
+}) => {
+  const BadgeClasses = classNames(
+    styles.shared,
+    styles[`variant-${color}`],
+    className
+  );
+  return <b className={BadgeClasses}>{children}</b>;
```

Comment:
The `b` <b> tag</b> default CSS  style   `font-weight:bold` but we need to customize it according the design.
  I think we can replace it with span or p tag. 
  
you can read this article form [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b)